### DISCUSSION TITLE
Need advice on clearing space on my pc

### DISCUSSION THREAD

**Discussion by DavidEdwardDarden** [nss-day-cohort-47/C47-Discussions#52] [Discuss a topic]
My computer throws me the error message: Low disk space You're running out of space on this PC. Manage storage to view usage and free up some space. So like usual I go and delete a few MB here and there... and then it pops up again... and this continues. The real REAL issue is I have 205 MB free on my C drive.... and 866GB free on my D drive. I have moved some things there in the past... but I am afraid I will move something vital and mess up my system. How do I determine what I can move to my D drive and what needs to stay on the C drive. Thanks.

I googled:
How do I determine what I can move to my D drive and what needs to stay on the C drive. The results where mostly... buy my cloud service here!

**Comment by Chadwell-Clark** [comment]
Do a search for files by size. Music files, videos and images can be moved without worry. Look through your downloads and get rid of as much as you can or move it to your D drive
Stay out of system folders and off your root

**Reply by coltmay** [reply]
To add to this.  I find that [WinDirStat](https://sourceforge.net/projects/windirstat/) is a pretty great program to help visual where your data lives and how much of it you have per directory or drive.  I use it occasionally.
